config/containers/logging/configure.md

```diff
@@ -43,32 +44,25 @@ example sets two configurable options on the `json-file` logging driver:
 {
   "log-driver": "json-file",
   "log-opts": {
-    "max-size": "10m",
-    "max-file": "3",
     "labels": "production_status",
     "env": "os,customer"
   }
 }
 ```
 
-> **Note**: `log-opt` configuration options in the `daemon.json` configuration
-> file must be provided as strings. Boolean and numeric values (such as the value
-> for `max-file` in the example above) must therefore be enclosed in quotes (`"`).
```

Comment:
And here; looks like a rebase issue / branch out of date

Reply:
Yeah, @thaJeztah - this was my quick attempt at producing a `diff` for what was merged into Amberjack vs. what lives in `master`.

Reply:
Wasn't sure if this was needed so left it out. Please let me know if this should be added back in.

Reply:
Crap - just realized that it probably needs to be included. Just added.